filter/src/main/resources/reference.conf

```diff
@@ -0,0 +1,89 @@
+// Reference configuration of metric filters and predefined default values.
+//
+// - Metrics are organized to a tree structure that may split on any level.
+// - Root node is virtual and defines defaults that may be later overwritten on any level.
+// - Branch nodes may be used to configure subtree of metrics, they may also hold metrics.
+// - Leaf nodes are metrics like counters and timers that carry values.
+// - Each metric may compute and provide fields like count, rate, mean, percentiles, etc.
+// - Configurations from all levels are merged together, the most specific one wins.
+// - Separator for node names in this configuration is '/', separator used in real is implementation specific.
+// - The underlying format and parsing library is https://github.com/typesafehub/config.
+//
+// root (virtual)
+// |
+// +--- name1 --- + --- name2 --- ... --- nameN --- + --- myTimer
+// |              |                                 + --- myMeter
+// |              + --- haf --- ...
+// +--- ...
+//
+metricFiltersDefaults {
+  // Virtual root filter that is always defined.
+  root {
+    // Enable/disable flag for metrics reporting.
+    //
+    // (Future versions may extend the set of possible values to enable, error, warn, info, debug, trace, disable.
+    // Incompatible API changes would be needed.)
+    level = enable
+
+    // Counter fields to be reported.
+    counter {
+      count = enable
+    }
+
+    // Gauge fields to be reported.
+    gauge {
+      value = disable
+    }
+
+    // Meter fields to be reported.
+    meter {
+      count = enable
+      mean = disable
+      oneMinuteRate = disable
+      fiveMinuteRate = disable
+      fifteenMinuteRate = disable
+    }
+
+    // Histogram fields to be reported.
+    histogram {
+      percentiles = [0.5, 0.99]
+      count = disable
+      min = disable
+      max = disable
+      mean = disable
+      stdDev = disable
+    }
+
+    // Timer fields to be reported.
+    timer {
+      percentiles = [0.5, 0.99]
```

Comment:
The range is 0 - 1. The behavior will be implementation specific - we can ignore it, use closest correct one or fail fast. Dropwizard throws IllegalArgumentException.